src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int leftLeg;
+    private int rightLeg;
+
+    public RightTriangle(String color, int leftLeg, int rightLeg) {
+        super(color);
+        this.leftLeg = leftLeg;
+        this.rightLeg = rightLeg;
+    }
+
+    @Override
+    public double area() {
+        return (leftLeg * rightLeg) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rightTriangle, area: " + area()
+                + " sq. units, left Leg: " + leftLeg
+                + " sq. units, right Leg: " + rightLeg
+                + " units, color: " + getColor());

```

Comment:
The `System.out.println()` statement here is not in English. Please use only English in your messages/code to make it universal and consistent. This is a code style issue.